graphs/templates/graphs/upload_graph.html

```diff
@@ -52,12 +52,13 @@ <h3 style="text-align: center; color: red;">Error: {{Error}}</h3>
   <label class="col-md-4 control-label" for="filebutton"></label>
   <div class="col-md-4" id="test">
     <span class="btn btn-default btn-file" id="filename">
-	    Supports uploading of files in two distinct formats (.cyjs) and (.json) <input id="graphname" name="graphname" type="file" accept=".cyjs,.json"><br>
+	    Supports uploading of files in (.cyjs), (.gpml) and (.json) formats <input id="graphname" name="graphname" type="file" accept=".cyjs,.json,.gpml"><br>
 	</span>
   <p id="files" style="color: red;"></p>
   </div>
 </div>
 <p style="font-size: 11px;text-align: center;color: green;">You can export a network created in Cytoscape (v3.1 or later) in this format. See <b><a href="{{url}}help/#upload_graphs">documentation</a></b> for more information.  Refer to the <a href="{{url}}help/programmers/#example">example JSON</a>.</p>
+<p style="font-size: 11px;text-align: center;color: green;"><b>NOTE:</b> Limited support for GPML format</p>
```

Comment:
@MridulS , I suggest the phrase "GPML support in GraphSpace has limitations." 

The page that @adbharadwaj refers to should be part of the GraphSpace documentation, e.g., at http://graphspace.org/help#upload_graphs_cyjs